Tweaks

simonwiles opened the pull request:
Some more tweaks; all subject to further discussion, if desired.

simonwiles committed:
Remove flat background from transport controls strip

simonwiles committed:
Use `--primary-accent` for transport controls hover style

This changes only the icon, leaving the border white on hover, as before.  I still don't love it, tbh, but I think it's unambiguously better than before.
update: added a background too, as without it this change didn't work too well with the grey theme.  I like it more this way, I think 🤷

simonwiles committed:
Move the mode switch in a bit

This is to counteract the impression of it being out of alignment because of the transparent gap on the left of (almost?) all the roll images.  The button now appears inset a little when the roll image is zoomed in (and covering all the space reserved for it), but that's not nearly so niggling as the apparent mis-alignment when the roll loads in "full-width".

simonwiles committed:
Adjust `<ViewerMetrics/>` styling

Controversial?  The margin was bothering me, and the background change was motivated by a desire to distinguish the panel from the buttons below.  You could make a case that the buttons should have the `primary` accent rather than the panel?

simonwiles committed:
Restore opacity for pedal button `<kbd/>`s

The hover styles were removed in 4d2f89c6, but this left the indicators forever in a state of limbo (or something).  Still feel like these buttons need a`:hover` state as well, tbh.

simonwiles committed:
Prevent left sidebar from showing through control strip when collapsed

broadwell commented:
I like everything so far -- the transport button hover styling is certainly an improvement, and bringing the `<ViewerMetrics>` "forward" in the layout really helps.
I agree that it's probably preferable for the pedal buttons to have the primary color styling, in which case the `<ViewerMetrics>` probably could just have the grey styling that's current applied to the buttons.
And I also concur that the pedal buttons need a hover style; the only stipulation I'd add is that it should be more subtle than their actual pressed/active style, because it's possible for each button to be either pressed or not pressed when hovered, and it's important to be able to ascertain its pressed status both when it's hovered and not.

simonwiles committed:
Add better hover and active styles to player controls

Also cleans up the duplication and unnecessary nesting, and adjusts the `<ViewerMetrics/>` styling.

simonwiles committed:
Make sure RollViewerControls buttons can be seen against the background

broadwell reviewed (approved):
This all looks great to me -- thanks!
